declarations_site/spotter/migrations/0002_searchtask_chat_data.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.5 on 2017-07-01 16:17
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('spotter', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='searchtask',
+            name='chat_data',
+            field=models.TextField(blank=True, default='', max_length=2000, verbose_name='Дані чату'),
```

Comment:
Not in this case, but im not sure that future channels will be always used only json tokens, so prefer keep common BLOB field